README.md

```diff
@@ -14,28 +14,27 @@
   - [Network Path](#network-path)
 - [Prerequisites](#prerequisites)
 - [Installation](#installation)
-  - [Installation using Operator via subctl](#installation-using-operator-via-subctl)
+  - [Installation using subctl](#installation-using-subctl)
   - [Installation using Helm](#installation-using-helm)
   - [Validate Submariner is Working](#validate-submariner-is-working)
 - [Building and Testing](#building-and-testing)
-- [Known Issues/Notes](#known-issuesnotes)
-  - [OpenShift Notes](#openshift-notes)
+- [Known Issues](#known-issues)
 - [Contributing](#contributing)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 Submariner is a tool built to connect overlay networks of different Kubernetes clusters. While most testing is performed against Kubernetes
-clusters that have enabled Flannel/Calico/Canal/Weave/OpenShiftSDN, Submariner should be compatible with any CNI cluster network
+clusters that have enabled Flannel/Calico/Canal/Weave/OpenShiftSDN, Submariner should be compatible with most CNI cluster network
 provider, as it utilizes off-the-shelf components to establish encrypted tunnels between each Kubernetes cluster.
 
-Note that Submariner is in the **pre-alpha** stage, and should not be used for production purposes. While we welcome usage and
+Note that Submariner is in an early stage, and should not be used for production purposes. While we welcome usage and
```

Comment:
```suggestion
Note that Submariner is in an early stage, and while we welcome usage and
```

Reply:
Folks are already using it in production, so let's remove this wording.

Reply:
Yeah, but should they be? I thought many of the goals of 0.12 were to get SubM ready for production, which would imply we don't think it's ready for production now.

Reply:
@skitt @sridhargaddam wdyt?

Reply:
What does “ready for production” mean for an open source project anyway? Submariner as it exists now works and is usable in many scenarios.

0.12 goals are more about supportability, _i.e._ our ability to help people who are running into difficulties using Submariner. There isn’t much going into 0.12 which improves the core features of Submariner itself in a significant way.

Reply:
I guess in one sense “ready for production” means we would recommend running it in production. Our work on supportability seems to be a prerequisite for us "officially" making that recommendation. Others prerequisites we've identified that come to mind include rotating certs and an automated/reliable uninstall flow.

Reply:
Maybe someone else (who feels more comfortable with that change) would like to send a PR for this, and it can be discussed independently?

Reply:
I sent a second commit dedicated to this change, and explained the reasoning in the commit message.

Reply:
Right, that matches my thoughts — there’s a difference between explicitly discouraging production use, explicitly encouraging it, and somewhere in between.